{
  "channeldata": {
    "activate.d": false,
    "binary_prefix": false,
    "deactivate.d": false,
    "description": null,
    "dev_url": "https://github.com/LuminosoInsight/python-ftfy",
    "doc_source_url": null,
    "doc_url": "http://ftfy.readthedocs.io/",
    "home": "https://github.com/LuminosoInsight/python-ftfy/",
    "icon_hash": null,
    "icon_url": null,
    "identifiers": null,
    "keywords": null,
    "license": "MIT",
    "post_link": false,
    "pre_link": false,
    "pre_unlink": false,
    "recipe_origin": null,
    "run_exports": {},
    "source_git_url": null,
    "source_url": "https://pypi.io/packages/source/f/ftfy/ftfy-4.4.2.tar.gz",
    "subdirs": [
      "osx-64"
    ],
    "summary": "Fixes some problems with Unicode text after the fact",
    "tags": null,
    "text_prefix": true,
    "timestamp": 0,
    "version": "4.4.2"
  },
  "channeldata_version": 1,
  "feedstock": null,
  "labels": [
    "main",
    "cf202003"
  ],
  "package": "ftfy-4.4.2-py35_0.tar.bz2",
  "repodata": {
    "build": "py35_0",
    "build_number": 0,
    "depends": [
      "html5lib",
      "python 3.5*",
      "setuptools",
      "wcwidth"
    ],
    "license": "MIT",
    "md5": "18ac3ac2621dd97996ab29c797a2ec39",
    "name": "ftfy",
    "sha256": "36d253415af70c980ec0255e434529f4f30c78f736073048cb60fb75ec91e8a8",
    "size": 51970,
    "subdir": "osx-64",
    "version": "4.4.2"
  },
  "repodata_version": 1,
  "subdir": "osx-64",
  "url": "https://conda.anaconda.org/conda-forge/osx-64/ftfy-4.4.2-py35_0.tar.bz2"
}